src/main/java/com/timgroup/statsd/NonBlockingStatsDClient.java

```diff
@@ -62,10 +62,10 @@
     private static final String ENTITY_ID_TAG_NAME = "dd.internal.entity_id" ;
 
     enum Literal {
-        service,
-        env,
-        version,
-        tags;
+        SERVICE,
+        ENV,
+        VERSION,
+        TAGS;
```

Comment:
What is the function of `TAGS`? Do we end up with a tag that has this sort of structure: `tags:<DD_TAGS>`?

Reply:
You right. Should I delete it ? I think we don't need it

Reply:
removed